Purchases/SwiftSources/Purchasing/TransactionsFactory.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PurchaserInfoHelper.swift
+//  Purchases
+//
+//  Created by RevenueCat.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCTransactionsFactory) public class TransactionsFactory: NSObject {
+
+    @objc public func nonSubscriptionTransactions(with subscriptionsData: [String: [[String: Any]]], dateFormatter: DateFormatter) -> [Transaction] {
```

Comment:
let's add a line break in between params to stick to 120 chars per line whenever possible 
```swift
     @objc public func nonSubscriptionTransactions(with subscriptionsData: [String: [[String: Any]]], 
                                                   dateFormatter: DateFormatter) -> [Transaction] {
```

Reply:
that `Any` can probably be constrained to `AnyObject`, right? since it's meant to be used from obj-c. 
The difference is that `Any` can also be non-objects (for example, a bool), whereas `AnyObject` means anything that's a class. 
This is a requirement for objective-c NSDictionaries anyway - if you want to insert a bool, you'd have to wrap it in an NSNumber *. 
https://medium.com/@mimicatcodes/any-vs-anyobject-in-swift-3-b1a8d3a02e00

Reply:
I forgot to delete that comment - after thinking about it more, I think it may not be worth it. It would make testing slightly trickier since you have to convert primitive types before calling the method, and it's already enforced for obj-c anyway. so.... nevermind. sorry for the noise.